spark/src/main/resources/error/delta-error-classes.json

```diff
@@ -1130,6 +1130,15 @@
     ],
     "sqlState" : "42K05"
   },
+  "DELTA_INVALID_CHARACTERS_IN_COLUMN_NAMES_UPON_SETTING_COLUMN_MAPPING_TO_NONE" : {
```

Comment:
Nit: Let's maybe use a slightly error class.

Reply:
Is `DELTA_INVALID_COLUMN_NAMES_WHEN_REMOVING_COLUMN_MAPPING` better?

Reply:
SGTM